examples/soundcloud.amp.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>Soundcloud examples</title>
+  <link rel="canonical" href="amps.html" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <link href='https://fonts.googleapis.com/css?family=Questrial' rel='stylesheet' type='text/css'>
+  <script async custom-element="amp-soundcloud" src="/dist/v0/amp-soundcloud-0.1.max.js"></script>
+  <style>body {opacity: 0}</style><noscript><style>body {opacity: 1}</style></noscript>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
```

Comment:
Here you are mixing local dist from the soundcloud embed with the production build. Make both production paths (like you head) and use the `.max.html` file generated into `examples.build` directory. Does that make sense?